docs/pages/setup/reference/config.mdx

```diff
@@ -89,6 +89,13 @@ teleport:
     # auth_token: /var/lib/teleport/tokenjoin
     auth_token: xxxx-token-xxxx
 
+    # join_params is used when joining a cluster via an ec2 or iam token
```

Comment:
I would probably find a way to mention that `auth_token` and `join_params` are mutually exclusive.